Content.Server/_NF/CryoSleep/CryoSleepSystem.cs

```diff
@@ -283,6 +283,14 @@ public bool InsertBody(EntityUid? toInsert, Entity<CryoSleepComponent> cryopod,
         return true;
     }
 
+    public bool BodyStorageSanityCheck(EntityUid body, Entity<CryoSleepComponent?> cryopod)
```

Comment:
```suggestion
    public bool CryopodContainsBody(EntityUid body, Entity<CryoSleepComponent?> cryopod)
```
Should use a more descriptive method name for this

Reply:
It's now "IsBodyInCryopod"